### PR TITLE
Add check data section for custom DC guide

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: custom_dc/prepare_data.md [hunk 1]
```diff
@@ -110,6 +112,13 @@ variableMeasured: Population
...
+
+Please check [Supported Date and Place
+Formats](https://datacommons.org/import/#supported-formats)
+
+## Check Data
 
-Please check [Supported Date and Place Formats](https://datacommons.org/import/#supported-formats)
+Before uploading Data into custom instance, make sure to run [Import
+Checker](https://github.com/datacommonsorg/import#using-import-tool) and make
+sure there is no format or other issues.
```
Comment (beets) [inline custom_dc/prepare_data.md:124]:
```suggestion
sure there are no formatting or other issues.
```

File: custom_dc/prepare_data.md [hunk 2]
```diff
@@ -110,6 +112,13 @@ variableMeasured: Population
...
-## Date and Place Formats
+### Date and Place Formats
+
+Please check [Supported Date and Place
+Formats](https://datacommons.org/import/#supported-formats)
+
+## Check Data
 
-Please check [Supported Date and Place Formats](https://datacommons.org/import/#supported-formats)
+Before uploading Data into custom instance, make sure to run [Import
```
Comment (beets) [inline custom_dc/prepare_data.md:122]:
```suggestion
Before uploading the data into custom instance, make sure to run [Import
```

File: custom_dc/prepare_data.md [hunk 3]
```diff
@@ -110,6 +112,13 @@ variableMeasured: Population
 value: C:data->Population
 ```
 
-## Date and Place Formats
+### Date and Place Formats
+
+Please check [Supported Date and Place
+Formats](https://datacommons.org/import/#supported-formats)
+
+## Check Data
```
Comment (beets) [inline custom_dc/prepare_data.md:120]:
```suggestion
## Testing Data
```
Testing, or Verifying or Validating data

**Review by beets — APPROVED** [review]

File: custom_dc/prepare_data.md (hunk 3 above)
Comment (shifucun) [reply custom_dc/prepare_data.md]: Done

File: custom_dc/prepare_data.md (hunk 2 above)
Comment (shifucun) [reply custom_dc/prepare_data.md]: Done

File: custom_dc/prepare_data.md (hunk 1 above)
Comment (shifucun) [reply custom_dc/prepare_data.md]: done

**Review by shifucun — COMMENTED** [review]
thanks for review!

**Pull request merged**